app/views/shared/_navbar.html.erb

```diff
@@ -8,8 +8,10 @@
           <li><%= link_to "Demo Nights", admin_demo_nights_path %></li>
           <% if !current_demo_night? %>
             <li><%= link_to "New Demo Night", new_admin_demo_night_path %></li>
+          <% else %>
+            <li><%= link_to "Current Projects", admin_demo_night_path(DemoNight.current) %></li>
           <% end %>
-          <li><%= link_to "Current Projects", admin_projects_path %></li>
```

Comment:
@s-espinosa What do you think about removing this view and replacing it with a redirect to the current Demo Night. I feel like these views should be one in the same. Is there an advantage to having just projects outside of the context of a demo night?